pages/linux/dunstify.md

```diff
@@ -0,0 +1,16 @@
+# dunstify
+
+> A notification tool that is an extension of notify-send, but has more features based around dunst.
+> Works with all options that work for notify-send.
```

Comment:
Could we add a more information link below this line?

Reply:
Added the command to get more information at the bottom!

Reply:
I only found a github page [here](https://github.com/dunst-project/dunst), not sure there is a online document outside.

Reply:
Yes, the parent project is dunst, the notification manager, but there is very little documentation about dunstify, which is the notification sending command

Reply:
No problem - thanks for checking @Jfeatherstone :smiley_cat: